newm-server/src/main/resources/application.conf

```diff
@@ -129,7 +129,7 @@ idenfy {
     sessionUrl = "https://ivs.idenfy.com/api/v2/token"
 
     signature {
-        Key = ${IDENFY_SIGNATURE_KEY}
+        key = ${IDENFY_SIGNATURE_KEY}
```

Comment:
I already fixed this typo in the Secret Manager. And doesn't affect the current version because we're still using ENV VAR.